QAchat/python/QAchat.ipynb

```diff
@@ -0,0 +1,428 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# BrainTrust QA Chat Tutorial\n",
+    "\n",
+    "<a target=\"_blank\" href=\"https://colab.research.google.com/github/braintrustdata/braintrust-examples/blob/main/\">\n",
+    "  <img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/>\n",
+    "</a>\n",
+    "\n",
+    "Welcome to [BrainTrust](https://www.braintrustdata.com/)! This is a quick tutorial on how to build and evaluate an AI question and answer chat assistant. The assistant answers questions based on the user's information that is already saved in a vector DB (Chroma).\n",
+    "\n",
+    "Before starting, make sure that you have a BrainTrust account. If you do not, please [sign up](https://www.braintrustdata.com) or [get in touch](mailto:info@braintrustdata.com). After this tutorial, learn more by visiting [the docs](http://www.braintrustdata.com/docs)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import json\n",
+    "import braintrust\n",
+    "import chromadb\n",
+    "import openai\n",
+    "from autoevals.string import *\n",
+    "from autoevals.llm import *\n",
+    "\n",
+    "PROJECTNAME = \"QAchatbot\"\n",
+    "OPENAI_API_KEY = \"YOURAPIKEY\"\n",
+    "BT_API_KEY=\"YOURAPIKEY\"\n",
```

Comment:
We should be consistent in all notebooks -- these shuold be `env` statements in a separate block